docs/development-guide/ReadWriteSplittingPluginPerformanceResults.md

```diff
@@ -0,0 +1,8 @@
+# Read-Write Splitting Plugin Performance Results
+
+## Benchmarks
+| Benchmark                                                     | Score       | Units |
+|---------------------------------------------------------------|-------------|-------|
+| No Plugin Enabled                                             | 1,105,514.4 | ms/op |
```

Comment:
1 million ms? Should we use a different time unit?

Reply:
1 million ms ?? that is 1000 seconds ... this surely can't be correct

Reply:
@davecramer the 1000 second numbers were the time it took to run the entire test 10 times. We should show the results for a single iteration so I divided that number by 10 to get results of ~110 seconds. Sergiy suggested below to change the results to show the overhead as the time difference between running the test with the plugin(s) and the baseline of no plugins enabled. Those numbers give a better understanding of the overhead caused by using the plugins